README.md

```diff
@@ -25,6 +25,15 @@ Mochatto is an open-source proximity voice chat application for virtual meetings
 1. `make dev`
 1. `make dev-up`
 
+
+#### Debugging Server
+
+Requires VSCode
+1. `make dev-up`
+2. open `mochatto/server` in VSCode
+3. Go to Debug tab
```

Comment:
I'm not sure if VSCode changed but for me it's:
Rub -> Start Debugging -> Docker: Attach to node (it should automatically be attached)
nbd though

Reply:
From nodejs documentation. Don't have any meaning for me
https://nodejs.org/en/docs/guides/debugging-getting-started/